cmd/olfullnode/devnet.go

```diff
@@ -600,7 +600,7 @@ func deployethcdcontract(conn string, nodeList []node) (*ethchain.ChainDriverOpt
 	lock_period := big.NewInt(25)
 
 	tokenSupplyTestToken := new(big.Int)
-	validatorInitialFund := big.NewInt(300000000000000000) //300000000000000000
+	validatorInitialFund := big.NewInt(0) //300000000000000000
```

Comment:
Is this needed?

Reply:
Should this PR be made from **MASTER** to the deployed taged version?